pilot/test/integration/routing.go

```diff
@@ -52,7 +52,7 @@ func (t *routing) run() error {
 			description: "routing all traffic to c-v1",
 			config:      "rule-default-route.yaml.tmpl",
 			check: func() error {
-				return t.verifyRouting("http", "a", "c", "", "", 100, map[string]int{"v1": 100, "v2": 0}, "default-route")
+				return t.verifyRouting("http", "a", "c", "", "", 100, map[string]int{"v1": 100, "v2": 0}, "route-rule-c")
```

Comment:
also, call it route-rule-a-to-cv1

Reply:
But then, why even change, since the name reflects the default route rule being applied

Reply:
Right now I am continually updating the v1alpha2 route rule named "route-rule-c". If we wanted to use different names, we would have to create and delete the rules.